hw_6/script.js

```diff
@@ -0,0 +1,89 @@
+//prettier-ignore
+let initialArray = [
+  47, 46, 48, 31, 63, 47, 36, 48, 36, 49, 50, 31, 50, 59, 63,
+  38, 35, 31, 50, 59, 63, 45, 36, 46, 38, 40, 35, 31, 45, 45,
+  58, 53, 63, 47, 46, 35, 31, 48, 42, 46, 33, 63, 46, 50, 63,
+  38, 40, 39, 45, 40, 63, 31, 63, 49, 31, 44, 46, 44, 51, 63,
+  35, 36, 43, 31, 50, 59, 63, 38, 40, 39, 45, 59
+];
+
+let mappedObject = initialArray.reduce((accumulator, char) => {
+  if (accumulator[char]) {
+    accumulator[char] += 1;
+  } else {
+    accumulator[char] = 1;
+  }
+  return accumulator;
+}, {});
+
+let sortedArray = Object.entries(mappedObject).sort(([, x], [, y]) => y - x);
+
+let ratio = null;
+
+// Object with correct ratio letter code with letters:
```

Comment:
It will be more clear, check out your code for this type of comment.
```suggestion
// Object with correct ratio letter code with letters:

/*ratio = {
  63: "  ",
  31: "а",
  45: "н",
  46: "о",
  50: "т",
  35: "д",
  36: "е",
  38: "ж",
  40: "и",
  59: "ь",
  47: "п",
  48: "р",
  39: "з",
  44: "м",
  49: "с",
  33: "в",
  42: "к",
  43: "л",
  51: "у",
  53: "х",
  58: "ы",
};*/
```

Reply:
https://github.com/BlackTruth/javascript/blob/26e480b6e9d9e22ba634d5513e6e430222b0590e/hw_6/script.js#L23-L48
Fixed.